console/console-init/ui/src/components/common/NetworkStatusAlert.tsx

```diff
@@ -26,6 +27,11 @@ const NetworkStatusAlert: React.FunctionComponent = () => {
     hasNetworkError !== undefined && setAlertVisible(hasNetworkError);
   }, [hasNetworkError]);
 
+  const errorMessage =
+    statusCode && statusCode === ErrorCodes.FORBIDDEN
+      ? "You're disconnected from the server, please sign in again."
```

Comment:
Strictly speaking, you are not disconnected from the server, the server is just refusing to let you in.  I think words to the effect of "Your session has expired. Please login again" would more accurately convey the problem.

```suggestion
      ? "Your session has expired. Please login again."
```

Reply:
Done, I have changed the message.